internal/output/output.go

```diff
@@ -122,8 +123,9 @@ func (o *Output) SetImageAccessibleCheckFromError(err error) {
 // SetImageSignatureCheck sets the passed and result.message fields of the ImageSignatureCheck to the given values.
 func (o *Output) SetImageSignatureCheckFromError(err error) {
 	metadata := map[string]interface{}{
-		"code":  "builtin.image.signature_check",
-		"title": "Image signature check passed",
+		"code":        "builtin.image.signature_check",
+		"title":       "Image signature check passed",
+		"description": "The image signature matches the provided public key.",
```

Comment:
It would be nice to phrase this in a way that it also makes sense for keyless signatures where a public key is not actually provided.

Reply:
@lcarva What do you think of "The image signature matches available signing materials"?

Reply:
Sounds good!